src/clinic.js

```diff
@@ -98,9 +98,16 @@ go.app = function() {
         };
 
         self.fire_incomplete = function(name, val) {
-            var ignore_states = ['states_end_success'];
+            var ignore_states = [];
             if (!_.contains(ignore_states, name)) {
-                return self.im.metrics.fire.inc(([self.metric_prefix, name, "no_incomplete"].join('.')), {amount: val});
+                return Q.all([
+                    self.im.metrics.fire.inc(
+                        ([self.metric_prefix, name, "no_incomplete_rev1"].join('.')), {amount: val}),
+                    self.im.metrics.fire.sum(
+                        ([self.metric_prefix, name, "no_incomplete_rev1.transient"].join('.')), val)
```

Comment:
I used transient to line up with the way jsbox-metricshelper names things: http://go-jsbox-metrics-helper.readthedocs.org/en/latest/dashboard.html#total-unique-users-and-sessions